packages/nc-gui/components/cell/Email.vue

```diff
@@ -70,8 +73,10 @@ watch(
     v-if="editEnabled"
     :ref="focus"
     v-model="vModel"
-    class="w-full outline-none text-sm px-1 py-2"
     :placeholder="isEditColumn ? $t('labels.optional') : ''"
+    class="w-full focus:!outline-none focus:ring-0 !border-none !outline-none text-sm px-2"
```

Comment:
I've left a comment on `!`. It seems the changes got overrode. Please double check.